rustls/src/crypto/mod.rs

```diff
@@ -588,6 +564,47 @@ pub fn default_fips_provider() -> CryptoProvider {
     aws_lc_rs::default_provider()
 }
 
+mod static_default {
+    #[cfg(not(feature = "std"))]
+    use alloc::boxed::Box;
+    use alloc::sync::Arc;
+
+    #[cfg(not(feature = "std"))]
+    use once_cell::race::OnceBox;
+    #[cfg(feature = "std")]
+    use once_cell::sync::OnceCell;
+
+    use crate::crypto::CryptoProvider;
```

Comment:
This could just be `super::CryptoProvider`.

Reply:
done